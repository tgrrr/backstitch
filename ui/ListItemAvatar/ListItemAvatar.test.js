import React from 'react'
import renderer from 'react-test-renderer'
import { ListItemAvatar } from '.'

it('ListItemAvatar: default', () => {
    const component = renderer.create(
        <ListItemAvatar>
            <>
                ListItemAvatar
            </>
        </ListItemAvatar>
    )
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
