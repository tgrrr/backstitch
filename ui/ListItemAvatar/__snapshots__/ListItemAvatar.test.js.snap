// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItemAvatar: default 1`] = `
<div
  className="sc-AxjAm StDqN"
>
  <div
    className="Listitemavatar "
    data-testid="Listitemavatar"
  >
    <div
      className="MuiListItemAvatar-root"
    >
      ListItemAvatar
    </div>
  </div>
</div>
`;
